src/unix/linux_like/linux/gnu/b64/powerpc64/mod.rs

```diff
@@ -190,6 +196,44 @@ s! {
         pub ss_flags: ::c_int,
         pub ss_size: ::size_t
     }
+
+    pub struct vscr_t {
+        if #[cfg(target_arch = "powerpc64")] {
+            __pad: [::c_uint; 3],
+            vscr_word: ::c_uint,
+        } else {
+            vscr_word: ::c_uint,
+            __pad: [::c_uint; 3],
+        }
+    }
+
+    pub struct vrregset_t {
+        pub vrregs: [[::c_uint; 4]; 32],
+        pub vscr: ::vscr_t,
+        pub vrsave: ::c_uint,
+        _pad: [::c_int; 3],
+    }
```

Comment:
glibc source: https://github.com/bminor/glibc/blob/master/sysdeps/unix/sysv/linux/powerpc/sys/ucontext.h#L110-L118

```c
/* Container for Altivec/VMX registers and status.
   Must to be aligned on a 16-byte boundary. */
typedef struct _libc_vrstate
{
	unsigned int	__ctx(vrregs)[32][4];
	vscr_t		__ctx(vscr);
	unsigned int	__ctx(vrsave);
	unsigned int	__pad[3];
} vrregset_t  __attribute__((__aligned__(16)));
```

Reply:
Hmm, the vrregset_t on CI totally mismatches it.

Reply:
Yup, what's your thought?